requirements.txt

```diff
@@ -10,3 +10,4 @@ psycopg2-binary
 geopandas
 aiofiles
 pytest-cov
+fastapi-auth0
```

Comment:
Hm, I'm not a big fan of introducing this dependency: [fastapi-auth0](https://pypi.org/project/fastapi-auth0/).
- maintained by someone not affiliated with Auth0
- last release in 12/'21 (>4 months ago)
- current stable version is `0.3.0` which seems very early
- 96 stars on GitHub is very low

Seeing as authentication is critical, paired with the points above, I'd be keen **not** to introduce this dependency into our production app.

I'd suggest that instead we [follow the official Auth0 guide here](https://auth0.com/blog/build-and-secure-fastapi-server-with-auth0/). What do you think?

Reply:
The guide there only does Bearing token ID, and doesnt do 0Auth with google login. I find the Google log in partiruclar useful. 

it was acknowledged by Auth0 though - https://community.auth0.com/t/using-auth0-with-fastapi/58764

i'm happy to pin the version to 0.3.0, so there are not security leaks in the future. 
I can also copy out the code from that repo - its only 200 lines. This seems the wrong way to do it though